lib/base/application.cpp

```diff
@@ -180,7 +180,7 @@ void Application::SetResourceLimits(void)
 	rlimit rl;
 
 #	ifdef RLIMIT_NOFILE
-	rl.rlim_cur = 16 * 1024;
+	rl.rlim_cur = GetRLimitFiles();
```

Comment:
No, this disables only setting of a stack limit, and it does not disable setting the limit. It sets the limit to MAX.

```
if (set_stack_rlimit)
```